app/DoctrineMigrations/Version20181219204218.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+namespace Pelagos\Migrations;
+
+use Doctrine\DBAL\Migrations\AbstractMigration;
+use Doctrine\DBAL\Schema\Schema;
+
+/**
+ * Auto-generated Migration: Please modify to your needs!
+ */
+class Version20181219204218 extends AbstractMigration
+{
+    /**
+     * @param Schema $schema
+     */
+    public function up(Schema $schema)
+    {
+        // this up() migration is auto-generated, please modify it to your needs
+        $this->abortIf($this->connection->getDatabasePlatform()->getName() !== 'postgresql', 'Migration can only be executed safely on \'postgresql\'.');
+
+        $this->addSql('ALTER TABLE dataset_submission_review DROP CONSTRAINT fk_3fa5c62f8488ba54');
+        $this->addSql('DROP INDEX uniq_3fa5c62f8488ba54');
+        $this->addSql('ALTER TABLE dataset_submission_review DROP dataset_submission_id');
```

Comment:
Considering adding SQL to populate the new field before loosing reference to the old one.

Reply:
I've added that update statement to the migration, thanks mike